back/lib/tasks/single_use/20250702_migrate_voting_term.rake

```diff
@@ -0,0 +1,27 @@
+namespace :single_use do
+  desc 'Configures the voting_term based on the existing voting_term_singular_multiloc and voting_term_plural_multiloc attributes of phases.'
+  task update_voting_term: :environment do
+    Tenant.safe_switch_each do |tenant|
+      puts "\nProcessing tenant #{tenant.host} \n\n"
+
+      map_to_term = {
+        'vote' => %w[vote stem stimme voto voorkeur],
+        'point' => %w[],
+        'token' => %w[banaan],
+        'credit' => %w[]
+      }
```

Comment:
@brentguf I leave it to you to complete this table. (the banaan was just a test I forgot to remove :D)

Reply:
@kogre Ok! Does this check all case variations of the terms, or should I add them exactly as they're used? We have "VOTE", "Vote", "stem", etc.

Reply:
It's a case insensitive comparison, so a single term suffices
```
 WHERE LOWER(value) = LOWER(?)
 ```

Reply:
Thanks, didn't check the code 🙃